docs/source/components/nodes/stereo_depth.rst

```diff
@@ -53,6 +53,16 @@ Inputs and Outputs
 Disparity
 #########
 
+Disparity refers to the distance between two corresponding points in the left and right image of a stereo pair.
+By looking at the image below, it can be seen that point :code:`X` gets projected to :code:`XL = (u, v)` in the :code:`Left view` and :code:`XR = (p, q)`in the :code:`Right view`.
```

Comment:
:code:`XR = (p, q)`in the :code:`Right view`.

^^ missing space after `XR = (p, q)` not rendered properly on the web